### DISCUSSION TITLE
Add viem plugin to wagmi cli

### DISCUSSION THREAD

**Discussion by roninjin10** [wevm/wagmi#2103] [General]
Is anybody working on this?  I'd like to add this so I can use this this weekend

**Comment by roninjin10** [comment]
@jxom @tmm is Viem plugin for wagmi cli unblocked now that contracts are in?

**Reply by jxom** [reply]
should be able to implement it if you want

**Reply by roninjin10** [reply]
Definitely I need it for quite a few things I'm doing soon
